plugins/publish-plugin/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+import org.gradle.accessors.dm.LibrariesForLibs
+
+val libs = the<LibrariesForLibs>()
+
+buildscript {
+  repositories {
+    google()
+    mavenCentral()
+  }
+}
+
+plugins {
+  `java-gradle-plugin`
+  `kotlin-dsl`
+  `kotlin-dsl-precompiled-script-plugins`
```

Comment:
Do we need it? AFAIK it is for `gradle.kts` files that you want to put into `src` folder.

Reply:
removed